plugins/council/README.md

```
@@ -1,11 +1,10 @@
 # council
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
-[![Skills](https://img.shields.io/badge/Skills-2-blue.svg)]()
+[![Skills](https://img.shields.io/badge/Skills-3-blue.svg)]()
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)